### ISSUE TITLE
Values containing arrays may be returned as string, causing errors (GP Media Library)

### ISSUE THREAD

**Issue by doekenorg** [GravityKit/GravityExport-Lite#152]
A recent support issue showed problems with fields that return an array of values; which are supposed to be a string. This problem is very clear on the GravityPerks Media Library plugin; where the media Id's field returns an array.

This issue is two-fold: 

- [ ] Type checks for (internal PHP) functions before call them
- [ ] Check if we can merge these values by default using the `glue`

As a feature we can also:
- [ ] Add support for the Media Library field by default

**Comment by doekenorg** [comment]
The temporary fix for the customer was:

```php
add_filter( 'gfexcel_meta_value', function ( $value, $entry, $field ) {
	if ( strpos( $field->id, 'gpml_ids_' ) !== false && is_array( $value ) ) {
		$value = implode( ", ", $value );
	}

	return $value;
}, 10, 3 );
```

**Comment by doekenorg** [comment]
Released in 2.0.0
